src/Compilers/VisualBasic/Portable/Compilation/MethodCompiler.vb

```diff
@@ -626,11 +626,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         If method.IsPartial() Then
                             Dim impl = method.PartialImplementationPart
                             If impl IsNot method Then
-                                If CType(method, SourceMethodSymbol).SetDiagnostics(ImmutableArray(Of Diagnostic).Empty) Then
-                                    method.DeclaringCompilation.SymbolDeclaredEvent(method)
-                                End If
+                                CType(method, SourceMethodSymbol).SetDiagnostics(ImmutableArray(Of Diagnostic).Empty)
```

Comment:
What is the purpose of calling SetDiagnostics with an empty array? Does it do something useful?